pkg/common/utils.go

```diff
@@ -0,0 +1,30 @@
+package common
```

Comment:
If you are deciding to keep exponential backoff, then please move this method to a better location. This file seems to be placed in a very top-level location.
If this continues to be a separate file, then add copyright header.